pom.xml

```diff
@@ -90,7 +90,7 @@
         <dependency>
             <groupId>com.funnyboyroks</groupId>
             <artifactId>DrawLib</artifactId>
-            <version>1.0.0-SNAPSHOT</version>
+            <version>1.0.2-SNAPSHOT</version>
         </dependency>
```

Comment:
The maven repo for this library isn't terribly reliable.  I've been wanting to switch this to use jitpack instead.

Would you go ahead and do that in this PR?

Should just involve replacing https://github.com/funnyboy-roks/worldedit-selection-viewer/blob/9e44489e1870071352b34615a66762933bb9ae8f/pom.xml#L72-L76 with
```xml
<repository>
    <id>jitpack.io</id>
    <url>https://jitpack.io</url>
</repository>
```

and replacing the dependency with 

```xml
<dependency>
    <groupId>com.github.funnyboy-roks</groupId>
    <artifactId>DrawLib</artifactId>
    <version>1.1.0-SNAPSHOT</version>
</dependency>
```